src/redux/books/books.js

```diff
@@ -1,7 +1,20 @@
 const ADD_BOOK = 'bookstore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
 
-const initialState = [];
+const initialState = [
+  {
+    title: 'The Nature of Nature',
+    author: 'Enric Sala',
+  },
+  {
+    title: 'The Design of Everyday Things',
+    author: 'Donald A. Norman',
+  },
+  {
+    title: 'The Art of War',
+    author: 'Sun-Tsu',
+  },
+];
```

Comment:
- [ ] Please remove the arrays and also on line 29 make sure to push directly to the booklist